Runtime/Scripts/BaseMeshSync.Textures.cs

```diff
@@ -146,7 +146,12 @@ partial class BaseMeshSync {
     /// <summary>
     /// Returns the default shader for the active render pipeline.
     /// </summary>
-    private static Shader GetStandardShader() {
+    private Shader GetStandardShader() {
+        if (GetConfigV().GetModelImporterSettings().DefaultShader != null)
```

Comment:
I guess we can cache `GetConfigV().GetModelImporterSettings().DefaultShader` directly into a local variable to reduce function calls.

Reply:
Done